lib/events/event_handler.ex

```diff
@@ -56,26 +59,45 @@ defmodule ElixirTools.Events.EventHandler do
     :ok
   end
 
+  @spec publish(Event.t(), event_schema, [events_opt]) :: :ok
+  def publish(event, schema, opts) do
```

Comment:
now I understand how both publish functions work together - `opts` for some strange reasons dont have default value `\\ []` as it usually has

Reply:
Uhm, your right!